scripts/live-connector-e2e/report.py

```
@@ -56,36 +96,165 @@ def load_results(results_dir: Path, *, roots: list[Path] | None = None) -> list[
+            root=path.parent,
+            connector=connector,
+            classification=classification,
+            baseline_version=baseline_version,
+            candidate_version=candidate_version,
+            baseline_status=baseline_status,
+            candidate_status=candidate_status,
+            evidence_error=evidence_error,
+            result_rows=result_rows,
+        )
+        if item.classification == "candidate_regression" and not item.evidence_error:
+            if not any(_matches_candidate_failure(row, item) for row in item.result_rows):
```

Ruff (0.16.1)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)